src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tBody = table.querySelector('tbody');
+
+const getSalary = (salary) => {
+  return salary.innerText.slice(1).split(',').join('');
+};
+
+const sortByText = (rowsArray, index) => {
+  rowsArray.sort((a, b) => a.cells[index].innerText
+    .localeCompare(b.cells[index].innerText));
+};
+
+const sortByAge = (rowsArray, index) => {
+  rowsArray.sort((a, b) => +a.cells[index].innerText
+    - +b.cells[index].innerText);
+};
+
+const sortBySalary = (rowsArray, index) => {
+  rowsArray.sort((a, b) =>
+    +getSalary(a.cells[index]) - +getSalary(b.cells[index])
```

Comment:
small comment - better to convert the value to number in `getSalary()` and return a number
```suggestion
    getSalary(a.cells[index]) - getSalary(b.cells[index])
```